src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -139,6 +139,31 @@ public IntervalMergingRule getIntervalMergingRule() {
         return intervalMergingRule;
     }
 
+    /**
+     * Returns the full set of traversal intervals specified on the command line, including parsed intervals without
+     * merging intervals specified by the user on the command line. This is an advanced use case
+     *
+     * NOTE: this currently does not account for interval exclusion arguments
+     *
+     * @param sequenceDict used to validate intervals
+     * @return the full set of intervals specified on the command line, without any merging performed
+     */
+    public List<SimpleInterval> getIntervalsWithoutMerging(final SAMSequenceDictionary sequenceDict ) {
+        if (getIntervalStrings().isEmpty() ) {
+            throw new GATKException("Cannot call getIntervalsWithoutMerging() without specifying intervals to include.");
```

Comment:
The regular `getIntervals()` method uses the reference dictionary to construct a set of intervals when `getIntervalStrings()` is empty, rather than throwing an exception. Should this method do the same for the sake of consistency? (ie., return `GenomeLocSortedSet.createSetFromSequenceDictionary(sequenceDict).toList()` in that case?)

Reply:
It could potentially, for the purposes of Depth of Coverage I do not need this functionality as it requires intervals be specified. If i need to change it perhaps I will switch it there? I would really like to get this in so Depth of Coverage can be looked at.